src/Console/Command/FixCommand.php

```diff
@@ -320,7 +320,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ->getReporter($resolver->getFormat())
         ;
 
-        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : 'txt' === $reporter->getFormat() ? $output : null;
+        $stdErr = $output instanceof ConsoleOutputInterface
+            ? $output->getErrorOutput()
+            : ('txt' === $reporter->getFormat() ? $output : null)
+        ;
```

Comment:
@GrahamCampbell , this PR is only for allowing me merging #1824 without waiting another week. I will merge it as soon as CI will be green.